### DISCUSSION TITLE
SparkDotnet

### DISCUSSION THREAD

**Discussion by GeorgeS2019** [dotnet/spark#1174] [General]
**status 15th July 2024:** Done, 3.5.1 is now available

https://github.com/dbeavon/sparkdotnet

Let us get all the necessary PR merges, upgrade to .NET 8 to this branch.

**Comment by dbeavon** [comment]
@GeorgeS2019 I can try to take PR's for a period of time but ultimately we need this project to be revitalized.

**Comment by GeorgeS2019** [comment]
@dbeavon 
I am not sure who here can revitalise this repos.
If the dotnet team is not responding.
